libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/CriResponseItem.java

```diff
@@ -27,6 +27,7 @@ public class CriResponseItem implements PersistenceItem {
     private String status;
     private long ttl;
     private List<String> featureSet;
+    private boolean reproveIdentity;
```

Comment:
What happens for existing records - will this just default to `false`?

Reply:
Yeah - if there's no value in DDB, it'll come back as `false`